domains/melody.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "phantonducthang",
+           "email": "phantonducthang@hotmail.com",
+           "discord": "1190624410630099046"
+        },
+    
+        "record": {
+            "CNAME": "melody.64342fb9f8-hosting.gitbook.io"
```

Comment:
site gives an error (This site can’t provide a secure connection)

Reply:
Thanks man, I will check it again.